images/router/haproxy/Dockerfile

```diff
@@ -8,7 +8,10 @@ FROM openshift/origin-haproxy-router-base
 ADD conf/ /var/lib/haproxy/conf/
 ADD reload-haproxy /var/lib/haproxy/reload-haproxy
 ADD bin/openshift /usr/bin/openshift
-RUN ln -s /usr/bin/openshift /usr/bin/openshift-router
+RUN ln -s /usr/bin/openshift /usr/bin/openshift-router && \
```

Comment:
Whatever we have to do here we should document for folks in our image creation user guides - this is a really common thing - "hey, don't run as root, just give yourself cap net bind"